packages/widget_toolkit_pin/lib/src/lib_pin_code_with_biometrics/blocs/pin_code_bloc.dart

```diff
@@ -52,6 +52,14 @@ class PinCodeBloc extends $PinCodeBloc {
     required this.autoPromptBiometric,
   }) {
     authenticated.connect().addTo(_compositeSubscription);
+    Rx.merge([
+      _$addDigitEvent.map((digit) {
```

Comment:
Do not remove the check for maximum pin length

Reply:
Added